2019/assets/sass/paper-kit/_carousel.scss

```diff
@@ -82,5 +82,8 @@
             padding-left: 3px;
         }
         }
```

Comment:
Remove unnecessary bracket

Reply:
but is necessary, this bracket close the "a" block.